docs/introduction/glossary.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Glossary"
```

Comment:
should we rename this to "Vocabulary"? since it's known better? {ex: I just learned what this word is from google}